### PR TITLE
added sql queries

### PR DESCRIPTION
**Pull request by majakudlicka**
relates to #26

### CODE REVIEW

File: database/database_queries.js [hunk 1]
```diff
@@ -14,5 +14,57 @@ data.getStudents = (term, callback) => {
...
+                             }
+                             callback(null, res.rows);
+                          });
+                      };
+
+
+data.postStudents = (object) => {
+  dbConnection.query(`INSERT INTO students(first_name, last_name, email, DOB,
+                      univ_school, job_cat, picture, bio, phone)
+                      VALUES ('${object.first_name}', '${object.last_name}', '${object.email}',
```
Comment (des-des) [inline database/database_queries.js]: Vulnerable to sql injection attack

Comment (des-des) [inline database/database_queries.js]: Look at how you have created the one above, this is safe as `pg` module will keep you safe

File: database/database_queries.js [hunk 2]
```diff
@@ -14,5 +14,57 @@ data.getStudents = (term, callback) => {
...
+                                console.log(err);
+                            }
+                            console.log('Data saved in the database');
+                        });
+                    };
+
+
+data.postResidents = (object) => {
+  dbConnection.query(`INSERT INTO residents(first_name, last_name, email, address, DOB,
+                      picture, bio, phone) VALUES ('${object.first_name}', '${object.last_name}', '${object.email}',
```
Comment (des-des) [inline database/database_queries.js]: same as above

File: database/database_queries.js [hunk 3]
```diff
@@ -14,5 +14,57 @@ data.getStudents = (term, callback) => {
...
+                              if (err) {
+                                console.log(err);
+                              }
+                              console.log('Data saved in the database');
+                          });
+                      };
+
+data.postJobs = (object) => {
+dbConnection.query(`INSERT INTO jobs(start_date, start_time, end_date, end_time, description,
+                    rate, resident_id) VALUES ('${object.start_date}', '${object.start_time}', '${object.end_date}',
```
Comment (des-des) [inline database/database_queries.js]: same as above

**Comment by majakudlicka** [comment]
Added this file to my other PR to avoid resolving merge conflicts with master on this branch

**Comment by des-des** [comment]
@majakudlicka you should drop a link to the other pr here so I can follow where this has gone

**Pull request closed without merging**